base/compiler/ssair/irinterp.jl

```diff
@@ -141,7 +146,7 @@ function reprocess_instruction!(interp::AbstractInterpreter, inst::Instruction,
             (; rt, effects) = abstract_eval_statement_expr(interp, stmt, nothing, irsv)
             add_flag!(inst, flags_for_effects(effects))
         elseif head === :invoke
-            rt, (nothrow, noub) = concrete_eval_invoke(interp, stmt, stmt.args[1]::MethodInstance, irsv)
+            rt, (nothrow, noub) = concrete_eval_invoke(interp, inst, irsv)
```

Comment:
```suggestion
            rt, (nothrow, noub) = abstract_eval_invoke(interp, inst, irsv)
```
and rename `maybe_concrete_eval` to `concrete_eval_invoke`?

Reply:
Renamed to `abstract_eval_invoke_inst`